packages/osd-ui-framework/Gruntfile.js

```diff
@@ -73,180 +64,6 @@ module.exports = function (grunt) {
   grunt.loadNpmTasks('grunt-babel');
   grunt.loadNpmTasks('grunt-contrib-clean');
   grunt.loadNpmTasks('grunt-contrib-copy');
-  grunt.registerTask('prodBuild', ['clean:target', 'copy:makeProdBuild', 'babel:prodBuild']);
-
-  grunt.registerTask('docSiteBuild', function () {
-    const done = this.async();
-
-    const serverCmd = {
-      cmd: isPlatformWindows ? '.\\node_modules\\.bin\\webpack.cmd' : './node_modules/.bin/webpack',
-      args: [
-        '-p',
-        '--config=doc_site/webpack.config.js',
-        '--devtool=null', // Prevent the source map from being generated
-      ],
-      opts: { stdio: 'inherit' },
-    };
-
-    const uiFrameworkServerBuild = new Promise((resolve, reject) => {
-      grunt.util.spawn(serverCmd, (error, result, code) => {
-        if (error || code !== 0) {
-          const message = result.stderr || result.stdout;
-
-          grunt.log.error(message);
-
-          return reject();
-        }
-
-        grunt.log.writeln(result);
-
-        resolve();
-      });
-    });
-
-    uiFrameworkServerBuild.then(done);
-  });
-
-  grunt.registerTask('docSiteStart', function () {
-    const done = this.async();
-    Promise.all([uiFrameworkWatch(), uiFrameworkServerStart()]).then(done);
-  });
-
-  grunt.registerTask('compileCssLight', function () {
```

Comment:
Should we be removing this? The compiled css files do seem to be used in [uiRenderMixin](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/legacy/ui/ui_render/ui_render_mixin.js#L119)

Reply:
Yes, good eye. When I first looked at this I glossed over the fact that the KUI css has to be compiled manually to the /dist directory every time there is a change. :/

Reply:
Refactored those tasks.